evap/staff/tests/test_views.py

```diff
@@ -1764,6 +1813,18 @@ def test_copy_contributors(self):
         new_contributor_count = UserProfile.objects.filter(contributions__evaluation=self.evaluation).count()
         self.assertEqual(new_contributor_count, original_contributor_count + UserProfile.objects.filter(contributions__evaluation=self.evaluation2).count())
 
+    def test_copy_replace_contributors(self):
```

Comment:
shouldn't this be `test_copy_contributors`?
then, the same situation as in `test_copy_participants` applies here as well.

Reply:
I dont thinl so, the first test is for just copying the contributors into the evaluation. The second test is for replacing the current contributors.